src/content/home/helxFeatures.mdx

```diff
@@ -0,0 +1,57 @@
+import GridItem from "../../components/sections/features";
+import { GridContainer } from "../../components/grid.js";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper"
+import applications from "../../images/home/applications-workflow-home.svg";
+import search from "../../images/home/complex-search-func-icon-home.svg";
+import openSource from "../../images/home/open-source-icon-home.svg";
+import scalable from "../../images/home/scalable-icon-home.svg";
+import flexible from "../../images/home/flexibility-icon-home.svg";
+import metadata from "../../images/home/metadata-driven-icon-home.svg";
+
+<MaxWidthWrapper backgroundColor="#F5F6F5" maxWidth="x1" gutterBottom>
+
+<GridContainer spacing={6} padding="2rem 4rem 2rem 8rem">
+  <GridItem
+    image={applications}
+    altText="Applications and Workflow Icon"
+    title="Applications and Workflow"
+    subheading="Choose from pre-loaded industry standard tools and apps or customize above and beyond to support your work"
+  />
```

Comment:
i think this is another opportunity where you can use markdown for the title and subheading. to address the font issues, i would make the title an h3 (using `###`), and add the styles`sx={{"& h3": { fontSize: "1.1rem" }, "& p": { fontSize: "0.9rem" },}}` to the `<Box/>` that'll house the markdown `{children}` in the `/components/sections/features.js` file